runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -251,10 +251,11 @@ class TR_J9SharedCacheServerVM: public TR_J9ServerVM
    virtual TR::CodeCache * getResolvedTrampoline(TR::Compilation *, TR::CodeCache* curCache, J9Method * method, bool inBinaryEncoding) override { return 0; }
    virtual intptrj_t methodTrampolineLookup(TR::Compilation *, TR::SymbolReference *symRef, void *callSite) override { TR_ASSERT(0, "methodTrampolineLookup not implemented for AOT");  return 0; }
    virtual TR::CodeCache * getDesignatedCodeCache(TR::Compilation *comp) override;
-   virtual void * getJ2IThunk(char *signatureChars, uint32_t signatureLength,  TR::Compilation *comp) override { TR_ASSERT(0, "called");  return 0; }
-   virtual void * setJ2IThunk(char *signatureChars, uint32_t signatureLength, void *thunkptr,  TR::Compilation *comp) override { TR_ASSERT(0, "called");  return 0; }
-   virtual void * persistJ2IThunk(void *thunk) override { TR_ASSERT(0, "called");  return 0; }
-   void * persistThunk(char *signatureChars, uint32_t signatureLength, uint8_t *thunkStart, uint32_t totalSize) { TR_ASSERT(0, "called");  return 0; }
+   virtual void * getJ2IThunk(char *signatureChars, uint32_t signatureLength,  TR::Compilation *comp) override;
+   virtual void * setJ2IThunk(char *signatureChars, uint32_t signatureLength, void *thunkptr,  TR::Compilation *comp) override;
+   virtual void * persistJ2IThunk(void *thunk) override;
+   void * persistThunk(char *signatureChars, uint32_t signatureLength, uint8_t *thunkStart, uint32_t totalSize);
```

Comment:
This is marked virtual in `TR_J9SharedCacheVM`. Is the absence of `virtual` on purpose? I do not see a strong reason to be virtual, unless we want to extend the hierarchy in the future.

Reply:
I was not meaning to drop `virtual`. I was removing `override` and accidentally deleted `virtual` as well. I think it's easier to make it virtual, just in case.